head/src/test/kotlin/io/qalipsis/core/head/jdbc/repository/DataSeriesRepositoryIntegrationTest.kt

```diff
@@ -162,6 +164,50 @@ internal class DataSeriesRepositoryIntegrationTest : PostgresqlTemplateTest() {
         assertThat(dataSeriesRepository.findAll().count()).isEqualTo(1)
     }
 
+    @Test
+    fun `should not save data-series with color opacity above 100`() = testDispatcherProvider.run {
```

Comment:
Please disable that test until we understand why the validation is not triggered here.